docs/reference/common-xref.md

```diff
@@ -0,0 +1,31 @@
+# Common dbxref and their shorthands
+
```

Comment:
I am not sure about this table.. If it is pre-scriptive, this needs more thought. Why would we refer to PRO, NCBITTaxon or ECO in two very different ways? Can you summarise the purpose for including this table?

Reply:
Personally, for me, its so I can easily grab what I need, but I did feel it a bit prescriptive, was hoping linking the top link and saying it is just common ones would do the trick. But happy to go with what @dosumis say and just link https://github.com/geneontology/go-site/blob/master/metadata/db-xrefs.yaml instead